vivisect/parsers/elf.py

```diff
@@ -166,6 +166,7 @@ def loadElfIntoWorkspace(vw, elf, filename=None, baseaddr=None):
     vw.addNoReturnApi("*.__assert_fail")
     vw.addNoReturnApi("*.__stack_chk_fail")
     vw.addNoReturnApi("*.pthread_exit")
+    vw.addNoReturnApi("*.longjmp")
```

Comment:
new noret

Reply:
where is it in PE?   we get to use "*." in ELF, but what library is it in PE, i can add it.